.github/workflows/llvm-project-epoch-three.yml

```diff
@@ -0,0 +1,32 @@
+name: llvm-project (epoch 3)
+
+on: workflow_dispatch
+jobs:
+  epoch3:
+    name: stage 3
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Build and test llvm-project
+        uses: ./.github/actions/build-test-llvm-project
+        with:
+          file: ./llvm-project/Dockerfile.epoch3
```

Comment:
This should be just `Dockerfile.epoch3`. The composite action prepends `./llvm-project/` because the action is only designed for `llvm-project/`.

Reply:
done PTAL